tests/parametric/test_span_links.py

```diff
@@ -42,60 +50,96 @@ def test_span_started_with_link(self, test_agent, test_library):
                     pass
 
         traces = test_agent.wait_for_num_traces(1)
-        trace = find_trace_by_root(traces, Span(name="root"))
-        assert len(trace) == 2
+        assert len(traces[0]) == 2
 
-        root = root_span(trace)
-        span = find_span(trace, Span(name="child"))
```

Comment:
nit: We do not need to use `root_span` and `find_span`. We can just access `traces` directly. I made this change in a few tests.